new cookbook: polydiapirism (#3004)
geodynamics/aspect: doc/manual/manual.tex

        effects might also play a role).
\end{itemize}

\subsubsection{Polydiapirism}
\label{sec:cookbooks-polydiapirism}

We consider a three-layer viscous Rayleigh-Taylor instability proposed by Weinberg and Schmeling \cite{wesc92} 
and we focus in what follows on the case II of this publlication.
The domain is a 2D Cartesian box of size $2.24\times 1$. Gravity is Earth-like.
Boundary conditions are free-slip on the sides and top and no-slip at the bottom.

> All three layers are initially horizontal. The top layer (fluid 1) has a thickness of 0.75, a viscosity $\eta_1=100$ and a density $\rho_1=100$. The middle layer (fluid 2) has a thickness 0.125 with $\rho_2=90$ and $\mu_2=1$.

Reviewer: Does the gravity have regular units (m/s^2)? Would it be possible to make all units either dimensional or non dimensional? Not critical to do so, but might be worth adding in sentence noting the difference.

We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.

\subsubsection{Polydiapirism}
\label{sec:benchmark-polydiapirism}

Diapirs are a type of geologic intrusion in which a more mobile and ductily deformable material (e.g. salt)
is emplaced into (brittle) overlying rocks. As salt domes are capable of trapping petroleum and 
natural gas these structures have been extensively studied \cite{jahu17}.

We consider in this experiment the three-layer viscous Rayleigh-Taylor instability proposed 
by Weinberg and Schmeling \cite{wesc92} 
and we focus in what follows on the case II of this publication.

> The domain is a 2D Cartesian box of size 2.24m$\times$1m. Gravity is Earth-like (9.81m/s$^2$).

Reviewer 2: Does this work or do you need a `{}` in front of the `^`?
```suggestion
The domain is a 2D Cartesian box of size 2.24m$\times$1m. Gravity is Earth-like ($9.81\text{m}/\text{s}^2$).
```

Boundary conditions are free-slip on the sides and top and no-slip at the bottom.
All three layers are initially horizontal. The top layer (fluid 1) has a thickness of 
0.75m, a viscosity $\eta_1=100$ Pa$\cdot$s and a density $\rho_1=100$kg/m$^3$. 
The middle layer (fluid 2) has a thickness 0.125m with $\rho_2=90$kg/$m^3$ and $\mu_2=1$Pa$\cdot$s.
The bottom layer (fluid 3) has a thickness 0.125m with $\rho_3=89$kg/m$^3$ and $\mu_3=1$Pa$\cdot$s.
The two interfaces between the layers are perturbed by a random noise of amplitude $\pm 0.001$m. Since fluid
3 is lighter than fluid 2 and fluid 2 is lighter than fluid 1, both interfaces are unstable.
We observe that interface 2-3 deforms first, produces domes which are subsequently incorporated in the domes 
being generated at the interface 1-2, as shown in Figure~\ref{fig:polydiapirs_density}.

> The root mean square velocity (Figure~\ref{fig:polydiapirs_vrms}) shows two slopes in the early stages (t$<$50s)

Reviewer 2: `t` is a variable, so ought to be shown in italics.
```suggestion
The root mean square velocity (Figure~\ref{fig:polydiapirs_vrms}) shows two slopes in the early stages ($t<50\text{s}$)
```

corresponding to the 
two different growth rates of the interfaces, 
as explained by linear stability analysis \cite{wesc92,ramb81}.

\begin{figure}
    \centering
    \includegraphics[width=0.48\textwidth]{cookbooks/polydiapirism/diapirs0000}
    \includegraphics[width=0.48\textwidth]{cookbooks/polydiapirism/diapirs0005}
    \includegraphics[width=0.48\textwidth]{cookbooks/polydiapirism/diapirs0010}
    \includegraphics[width=0.48\textwidth]{cookbooks/polydiapirism/diapirs0015}

>     \caption{Density field at t=0,25,50,75s}

Reviewer 2: ```suggestion
    \caption{Polydiapirism benchmark: Density field at $t=0,25,50,75\text{s}$.}
```

    \label{fig:polydiapirs_density}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.75\textwidth]{cookbooks/polydiapirism/vrms}

>     \caption{Root mean square velocity as a function of time}

Reviewer 2: ```suggestion
    \caption{Polydiapirism benchmark: Root mean square velocity as a function of time.}
```

\subsubsection{Polydiapirism}
\label{sec:benchmark-polydiapirism}

\textit{This section was contributed by Cedric Thieulot.}

Diapirs are a type of geologic intrusion in which a more mobile and ductily deformable material (e.g., salt)
is emplaced into (brittle) overlying rocks. As salt domes are capable of trapping petroleum and 
natural gas these structures have been extensively studied \cite{jahu17}.

We consider in this experiment the three-layer viscous Rayleigh-Taylor instability proposed 
by Weinberg and Schmeling \cite{wesc92} 
and we focus in what follows on the case II of that publication.
The domain is a 2D Cartesian box of size 2.24m$\times$1m. Gravity is Earth-like ($9.81\text{m}/\text{s}^2$).
Boundary conditions are free-slip on the sides and top and no-slip at the bottom.
All three layers are initially horizontal. The top layer (fluid 1) has a thickness of 

> 0.75m, a viscosity $\eta_1=100$ Pa$\cdot$s and a density $\rho_1=100$kg/m$^3$. 

Reviewer 2: One more of these:
```suggestion
0.75m, a viscosity $\eta_1=100$ Pa$\cdot$s and a density $\rho_1=100 \text{kg}/\text{m}^3$. 
```
There might be more in this section I'm missing right now. I believe I remember that just having `$^3$` leads to error messages in the programs that convert latex into HTML. @tjhei wants to use these to make the manual more easily accessible.

> The middle layer (fluid 2) has a thickness 0.125m with $\rho_2=90$kg/$m^3$ and $\mu_2=1$Pa$\cdot$s.

Reviewer 2: ```suggestion
The middle layer (fluid 2) has a thickness 0.125m with $\rho_2=90 \text{kg}/\text{m}^3$ and $\mu_2=1$ \text{Pa}\cdot\text{s}$.
```

Reviewer 2: same in the next line.
